src/Neo/SmartContract/Native/PolicyContract.cs

```diff
@@ -73,6 +78,7 @@ internal override ContractTask Initialize(ApplicationEngine engine, Hardfork? ha
                 engine.Snapshot.Add(CreateStorageKey(Prefix_FeePerByte), new StorageItem(DefaultFeePerByte));
                 engine.Snapshot.Add(CreateStorageKey(Prefix_ExecFeeFactor), new StorageItem(DefaultExecFeeFactor));
                 engine.Snapshot.Add(CreateStorageKey(Prefix_StoragePrice), new StorageItem(DefaultStoragePrice));
+                engine.Snapshot.Add(CreateStorageKey(Prefix_AttributeFee).Add((byte)TransactionAttributeType.NotaryAssisted), new StorageItem(DefaultNotaryAssistedAttributeFee));
```

Comment:
If is not possible to query this value, why we need it?

Reply:
> If is not possible to query this value,

It should be possible, exactly like for other defined attributes (via `GetAttributeFee`).

> why we need it

We need an initial value of 0.1 GAS according to the [proposal](https://github.com/neo-project/neo/issues/1573#issuecomment-704874472) and [calculations](https://github.com/nspcc-dev/neo-go/issues/1545#issue-743844176), and after that the NotaryAssisted service fee per key obeys standard attribute price setting pipeline.

Reply:
I understand, this will ensure that the default value is not Zero. It must be checked also in `SetAttributeFee` ? maybe we need a minimum value per attribute?

Reply:
Well, currently it's not planned to have minimum constraint for this value, so it can be 0 for some special networks. But we may discuss a lower bound for it. An upper bound is the same as for any other attribute (10 GAS).

Reply:
I think it can be left up to committee decision. If committee wants zero -- OK, let it be zero, just like almost any other setting.